app/models/event.rb

```diff
@@ -0,0 +1,60 @@
+class Event
+
+  attr_reader :name, :start, :end, :description, :location, :link, :thumbnail
+
+  def initialize(sum, start_dt, end_dt, loc, desc, link, thumb)
+    @name = sum
+    @start = start_dt
+    @end = end_dt
+    @location = loc
+    @description = desc
+    @link = link
+    @thumbnail = thumb
+  end
+
+  def self.pull_calendar_data
+    #Create dynamic parameters for the API call
+    #See Google Calendar.List API Documentation for more parameters
+    params = { query: { key: ENV['GOOGLE_API_KEY'], 
+               calendarId: 'PantherHackers@gmail.com', 
+               timeMin: (Time.now - 1.month).iso8601, 
+               timeMax: (Time.now + 1.month).iso8601 }
+    }
+
+    response = HTTParty.get('https://www.googleapis.com/calendar/v3/calendars/calendarId/events', params)
+
+    event_array = response['items'].map { |event| 
```

Comment:
You are using the `.map { ... }` syntax. For multiline stuff, use `.map do |event|` and close the block with `end` on a different line.

e.g.
```ruby
response['items'].map do |event|
...
...
...
end
```